tests/integration/tasks/test_sql_server.py

```diff
@@ -49,3 +47,17 @@ def test_sql_server_create_table(caplog):
             credentials=json.loads(credentials_str),
         )
         assert "Successfully created table" in caplog.text
+
+
+def test_sql_server_to_df():
+    task = SQLServerToDF(config_key="AZURE_SQL")
+    df = task.run(query=f"SELECT * FROM {SCHEMA}.{TABLE}")
+    assert df.columns.to_list() == [
```

Comment:
Isn't it `tolist()`? Didn't run the tests yet but I think that's the name of this method.

Reply:
I think both versions are correct